Contact/IContactRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nate.ContactApp
+{
+    interface IContactRepository
+    {
+        void Post(ContactRepository database, Contact newContact);
```

Comment:
typically a Post will return an object.  if anything the caller is going to need to get the id of the record that was inserted into the database.